vgen/src/main/java/io/gapi/vgen/go/GoDiscoveryContext.java

```diff
@@ -130,4 +131,20 @@ public boolean isPageStreaming(Method method) {
     }
     return hasPageToken && hasNextPageToken;
   }
+
+  private static final ImmutableTable<String, String, String> API_VERSION_RENAME =
+      ImmutableTable.<String, String, String>builder()
+          .put("clouduseraccounts", "beta", "v0.beta")
+          .build();
+
+  // TODO(pongad): Find a more sustainable solution to this.
```

Comment:
It reads funny to have this before the main comment. Maybe move it at the end of the javadoc?